webpack.config.js

```diff
@@ -1,25 +1,23 @@
 const path = require('path')
-const webpack = require('webpack')
-
-const mode = process.env.NODE_ENV || 'production'
 
 module.exports = {
   output: {
-    filename: `worker.${mode}.js`,
+    filename: 'worker.js',
     path: path.join(__dirname, 'dist'),
   },
-  mode,
+  devtool: 'cheap-module-source-map',
```

Comment:
I missed something here, I believe you need `entry: path.join(__dirname, 'dist/worker.js')` or similar.

@koeninger this is why contributing to my PR rather than creating a new one is preferable, I could have fixed this without any input from you.

Reply:
at least without that I was getting errors while importing / processing webpack modules.

Reply:
**Update:** `entry` has nothing to do with it, it's to do with trying to upgrade to many things, I'll comment on the line below with what needs changing.